modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Chestnut Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Chestnut Club';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bronze Spear';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bronze Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bronze Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Brass Spear';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bouncer Club (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Boomerang';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Book Holder';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bone Arrow';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bolt Belt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bloody Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bloody Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Blind Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Blind Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bay Aquarium';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bastokan Crossbow';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bamboo Fishing Rod';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bamboo Fishing Rod';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Staff';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Staff (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Pole';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Pole (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Club';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Club (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Clogs';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Arrowwood Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Arrowwood Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Angel\'s Flute';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Angel\'s Flute (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ancient Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ancient Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acid Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acid Bolt';
+UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acheron Shield' AND ID = 54538;
```

Comment:
I think this is ToAU...

Reply:
Allakhazam has a April 2004 entry for it at https://web.archive.org/web/20040402131137/https://ffxi.allakhazam.com/db/item.html?fitem=4452
This would make it predate COP and make it Zilart. COP release was Sep. 14, 2004.

Doesn't that mean that some of these recipes should have a blank ContentTag to signify that they are the base game / Zilart?

Reply:
Oh wow. Nice find!

And yes, I think technically there will be some here that are 'COP' in this pull request that should be in the base game. I think if another private server needs that (ie, base/RoZ only) then they should be on the hook for auditing that in the short-term since we have to be a little more thoughtful in what we tackle before Horizon launch, but you're absolutely right that a lot of these will want to have RoZ or NULL tags.